apps/web/src/lang/fr.json

```diff
@@ -8136,20 +8136,182 @@
+  },
+  "DXtgrn": {
+    "defaultMessage": "Révisez votre candidature et envoyez-la",
+    "description": "Link text for the application review page."
+  },
+  "A9Urm+": {
+    "defaultMessage": "Lorsque vous envoyez votre candidature, une copie de votre profil est automatiquement créée.",
+    "description": "List item note 1 for the application review page."
+  },
+  "GXmpsW": {
+    "defaultMessage": "Avant de poursuivre, il est important que vous compreniez les remarques suivantes concernant votre candidature :",
+    "description": "Starting message for the application review page."
+  },
+  "JxDtt+": {
+    "defaultMessage": "Révisez votre candidature",
+    "description": "Page title for the application review page."
+  },
+  "MZrE40": {
+    "defaultMessage": "Vous avez dit répondre aux <strong>exigences minimales d’expérience ou de formation (deux années d’études postsecondaires)</strong>  et posséder l’expérience professionnelle suivante :",
+    "description": "Message on education requirements card on the application review page."
+  },
+  "O83lYZ": {
+    "defaultMessage": "Cette section résume votre CV tel que le verront les responsables du recrutement pour ces postes.",
+    "description": "Blurb for resume section of the application review page."
+  },
+  "Sd8883": {
+    "defaultMessage": "Vous avez répondu aux questions suivantes :",
+    "description": "Message in screening questions section of the application review page."
+  },
+  "T8eOrg": {
+    "defaultMessage": "Votre nom complet",
+    "description": "Label displayed for signature input in sign and submit section of application review page."
+  },
+  "PDbqFR": {
+    "defaultMessage": "\"J’ai relu tout le contenu de ma candidature\"",
+    "description": "Review list item for sign and submit section of application review page."
+  },
+  "V9lxDE": {
+    "defaultMessage": "Il semble que vous n’ayez pas encore répondu aux questions de sélection.",
+    "description": "Null state message in screening questions section of the application review page."
+  },
+  "bO9PB4": {
+    "defaultMessage": "J’envoie ma candidature"
+  },
+  "8BR20I": {
+    "defaultMessage": "Nous vous encourageons à tenir votre profil à jour, car c’est sa version actualisée qui servira lors des étapes à venir du processus d’embauche.",
+    "description": "List item note 3 for the application review page."
+  },
+  "pH8wF2": {
+    "defaultMessage": "Bravo, c’est réussi! En signant, vous confirmez ce qui suit :",
+    "description": "Instructions for sign and submit section of application review page."
+  },
+  "Pasf+O": {
+    "defaultMessage": "Les mises à jour de votre profil après l’envoi de votre candidature ne se refléteront pas dans votre dossier de candidature.",
+    "description": "List item note 2 for the application review page."
+  },
+  "fhgZRX": {
+    "defaultMessage": "Je signe et j’envoie ma candidature",
```

Comment:
_Je signe et j’envoie ma candidature_ is not the same as _Sign and submit_ in English.

Reply:
@gobyrne thoughts on this translation?

Reply:
What's the English? Screenshot maybe too?

Reply:
Oh, it's just sign and submit. If "submit" is for the whole application, honestly, this translation is not terrible.

Reply:
## Screenshot
![Screen Shot 2023-05-30 at 11 24 23](https://github.com/GCTC-NTGC/gc-digital-talent/assets/3046459/47c1524d-14d1-4cfc-804a-318621e2f115)

## Design
[Figma](https://www.figma.com/file/BXYVsu1fo9LS2IySJDUINY/Applicants---Job-application?type=design&node-id=636-59839&t=PX2EVHydukpOfKZr-0)

## Code
https://github.com/GCTC-NTGC/gc-digital-talent/blob/b8c8e92abbc6ccd02927927d2e32970e120e55e1/apps/web/src/pages/Applications/ApplicationReviewPage/ApplicationReviewPage.tsx#L434-L441